tools/devops/automation/templates/build/publish-html.yml

```diff
@@ -118,7 +118,82 @@ steps:
       $apiDiffData | ConvertTo-Json | Write-Host
       Write-Host "##vso[task.setvariable variable=APIDIFF_JSON_PATH]$path"
     displayName: 'Create API from stable diff gists'
-    timeoutInMinutes: 1
+    timeoutInMinutes: 5
+    env:
+      GITHUB_TOKEN: $(GitHub.Token)
+
+- ${{ if eq(parameters.runTests, true) }}:
```

Comment:
No needed if, looks like we already check it runTests is true in the line 42. Why adding it again?

Reply:
Good call! Copy and pasted and did not notice haha